README.md

```diff
@@ -259,7 +259,65 @@ BundleKeys.endpoints := Map("akka-remote" -> Endpoint("tcp"))
 
 In the above, no declaration of `services` is required as akka remoting is an internal, cluster-wide TCP service.
 
-## play[23|24|25]-conductr-bundle-lib
+## play25-conductr-bundle-lib
```

Comment:
Is it possible to have tabs in GitHub doc? I'd rather that the user decide which code sample. I'd rather not duplicate the readme (your comment can also apply to the other sections).